config/gcp.spc

```diff
@@ -20,6 +20,9 @@ connection "gcp" {
   # If not set, no impersonation is done.
   #impersonate_service_account = "YOUR_SERVICE_ACCOUNT"
 
+  # `ignore_error_messages` (optional) - List of additional GCP error message pattern to ignore for all queries.
```

Comment:
Is an error ignored if the message matches a value from here OR if the error code matches the other config arg?

Reply:
If the `ignore_error_messages` has been set and the pattern matches with the error message the the error will be ignored from here.